disnake/ext/commands/common_bot_base.py

```diff
@@ -395,6 +395,14 @@ def cogs(self) -> Mapping[str, Cog]:
         """Mapping[:class:`str`, :class:`Cog`]: A read-only mapping of cog name to cog."""
         return types.MappingProxyType(self.__cogs)
 
+    @property
+    def listeners(self) -> Mapping[str, List[CoroFunc]]:
```

Comment:
While a property does make sense, using a `get_listeners` method instead for consistency with cogs would be better